src/Http/Http.Results/src/TypedResults.cs

```diff
@@ -110,7 +128,25 @@ public static ContentHttpResult Content(string? content, string? contentType = n
     /// <param name="contentType">The content type (MIME type).</param>
     /// <param name="contentEncoding">The content encoding.</param>
     /// <returns>The created <see cref="ContentHttpResult"/> object for the response.</returns>
-    public static ContentHttpResult Text(string? content, string? contentType = null, Encoding? contentEncoding = null)
+    public static ContentHttpResult Text(string? content, string? contentType, Encoding? contentEncoding)
+        => Text(content, contentType, contentEncoding, null);
+
+    /// <summary>
+    /// Writes the <paramref name="content"/> string to the HTTP response.
+    /// <para>
+    /// This is an alias for <see cref="Content(string, string?, Encoding?, int?)"/>.
```

Comment:
```suggestion
    /// This is an alias for <see cref="Content(string?, string?, Encoding?, int?)"/>.
```

We need to fix up the other "This is an alias for" doc comments that weren't updated in this PR.

Also, which one is the alias? `Text` or `Content`? It's a little weird to consider them aliases to each other. Maybe we should just call them "equivalent".

```suggestion
    /// This is an equivalent to <see cref="Content(string?, string?, Encoding?, int?)"/>.
```

Reply:
OK, I'll make an executive decisions that `Text` is an alias to `Content` and update accordingly.